src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -79,55 +78,29 @@ const ChangeRequestsOverview: React.FC = () => {
     )
     .sort((a, b) => (a.dateReviewed && b.dateReviewed ? b.dateReviewed.getTime() - a.dateReviewed.getTime() : 0));
 
-  const displayCRCards = (crList: ChangeRequest[]) => (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        flexWrap: 'wrap',
-        overflow: 'auto',
-        justifyContent: 'flex-start',
-        '&::-webkit-scrollbar': {
-          height: '20px'
-        },
-        '&::-webkit-scrollbar-track': {
-          backgroundColor: 'transparent'
-        },
-        '&::-webkit-scrollbar-thumb': {
-          backgroundColor: theme.palette.divider,
-          borderRadius: '20px',
-          border: '6px solid transparent',
-          backgroundClip: 'content-box'
-        }
-      }}
-    >
-      {crList.map((cr: ChangeRequest) => (
-        <ChangeRequestDetailCard changeRequest={cr}></ChangeRequestDetailCard>
-      ))}
-    </Box>
-  );
+  const crToReviewRow: ChangeRequestRow = {
+    title: 'To Review',
+    crList: crToReview,
+    emptyMessage: 'No change requests to review'
+  };
+  const crUnreviewedRow: ChangeRequestRow = {
+    title: 'My Un-reviewed Change Requests',
+    crList: crUnreviewed,
+    emptyMessage: 'No un-reviewed change requests'
+  };
 
-  const renderChangeRequests = (title: string, crList: ChangeRequest[], emptyMessage: string) => {
-    return (
-      <>
-        <Typography variant="h5" gutterBottom>
-          {title}
-        </Typography>
-        {crList.length > 0 ? (
-          <Grid container>{displayCRCards(crList)}</Grid>
-        ) : (
-          <Typography gutterBottom>{emptyMessage}</Typography>
-        )}
-      </>
-    );
+  const crApprovedRow: ChangeRequestRow = {
+    title: 'My Recently Approved Change Requests',
+    crList: crApproved,
+    emptyMessage: 'No recently approved change requests'
   };
 
+  const overviewRowList = [crToReviewRow, crUnreviewedRow, crApprovedRow];
+
   return (
-    <Box>
-      {showToReview && renderChangeRequests('To Review', crToReview, 'No change requests to review')}
-      {renderChangeRequests('My Un-reviewed Change Requests', crUnreviewed, 'No un-reviewed change requests')}
-      {renderChangeRequests('My Recently Approved Change Requests', crApproved, 'No recently approved change requests')}
-    </Box>
+    <>
+      <Box>{showToReview && CRRow(overviewRowList)}</Box>
```

Comment:
Also the && logic will have problems here.  If showToReview is true, then the CRRow component will be rendered. If showToReview is false, then nothing will be rendered including the approved and unReviewed CRs.

Reply:
There are examples where something is pushed (appended to a list) on a conditional basis